project/Dependencies.scala

```diff
@@ -9,7 +9,7 @@ object Dependencies {
   //libraries
   val sentryRavenLogback = "io.sentry" % "sentry-logback" % "1.7.5"
   val scalaUri = "io.lemonlabs" %% "scala-uri" % "2.2.2"
-  val identityAuthPlay = "com.gu.identity" %% "identity-auth-play" % "3.239-C2"
+  val identityAuthPlay = "com.gu.identity" %% "identity-auth-play" % "3.248"
```

Comment:
C2 sounds funny for the old one - maybe that was the play 2.7 backwards compatible version!

Reply:
Yeah could be